Formula/u/uhubctl.rb

```diff
@@ -0,0 +1,24 @@
+class Uhubctl < Formula
+  desc "USB hub per-port power control"
+  homepage "https://github.com/mvp/uhubctl"
+  url "https://github.com/mvp/uhubctl/archive/refs/tags/v2.6.0.tar.gz"
+  sha256 "56ca15ddf96d39ab0bf8ee12d3daca13cea45af01bcd5a9732ffcc01664fdfa2"
+  license "GPL-2.0-only"
+  head "https://github.com/mvp/uhubctl.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
```

Comment:
No need to add this, it is the default already

Reply:
Thanks, I will remove it

Reply:
removed